lib/include/private/aws_iot_defender_internal.h

```diff
@@ -329,13 +336,27 @@ IotMqttError_t AwsIotDefenderInternal_MqttConnect( void );
  * Subscribe accept/reject defender topics.
  */
 IotMqttError_t AwsIotDefenderInternal_MqttSubscribe( IotMqttCallbackInfo_t acceptCallback,
-                                                     IotMqttCallbackInfo_t rejectCallback );
+                                                     IotMqttCallbackInfo_t rejectCallback,
```

Comment:
These function do not follow the naming guidelines for internal functions.

Also, these structs (IotMqttCallbackInfo_t) should not be passed by value.